scripts/dts/extract/default.py

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (c) 2018 Bobby Noelte
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from extract.globals import *
+from extract.directive import DTDirective
+
+##
+# @brief Manage directives in a default way.
+#
+class DTDefault(DTDirective):
+
+    def __init__(self):
+        pass
+
+    ##
+    # @brief Extract directives in a default way
+    #
+    # @param node_address Address of node owning the clockxxx definition.
+    # @param yaml YAML definition for the owning node.
+    # @param prop property name
+    # @param names (unused)
+    # @param[out] defs Property definitions for each node address
+    # @param def_label Define label string of node owning the directive.
+    #
+    def extract(self, node_address, yaml, prop, names, defs, def_label):
```

Comment:
`defs` is now global (defined in globals.py) , remove from parameters list.
Apply everywhere

Reply:
I know that defs is in globals. But the interface was kept. See current version of extract_dts_inlcudes.py. Do you really wan´t this new interface in a refcator only PR?

Reply:
ok, my bad. This was done in my local branch bu not merged actually.
You can discard this comment